.eslintrc

```diff
@@ -87,6 +90,22 @@
         "@shopify/jsx-no-hardcoded-content": "off",
         "@shopify/react-initialize-state": "off"
       }
+    },
+    {
+      "files": ["polaris.shopify.com/**/*"],
+      "extends": "plugin:@next/eslint-plugin-next/recommended",
+      "rules": {
+        "react/react-in-jsx-scope": "off",
+        "@shopify/jsx-no-hardcoded-content": "off",
+        "import/no-default-export": "off"
+      }
+    },
+    {
+      "files": ["polaris.shopify.com/next-env.d.ts"],
```

Comment:
This fill seems to get auto regenerated on build, so it seems we can't drop the eslint-disable lines directly in the file